Processor.php

```diff
@@ -4,16 +4,38 @@
 
 use Composer\IO\IOInterface;
 use Symfony\Component\Yaml\Inline;
-use Symfony\Component\Yaml\Parser;
-use Symfony\Component\Yaml\Yaml;
 
 class Processor
 {
     private $io;
 
+    /**
+     * @var <FileProcessorInterface> 
```

Comment:
NetBeans support it. I guess it's not a problem to change it to: `FileProcessorInterface[]`